omnigibson/metrics/energy_metric.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+
+from omnigibson.metrics.metrics_base import BaseMetric
+
+
+class EnergyMetric(BaseMetric):
+    """
+    Energy Metric
+
+    Measures displacement * mass for every link
+
+    Args:
+        measure_work: If true, measure beginning and end delta rather than step by step delta
+    """
```

Comment:
I think we should just create a new WorkMetric instead of combining both into the same class with a boolean flag